constants/exploredata.js

```diff
@@ -1,4 +1,4 @@
-export const categories = ["Food", "Shopping", "Markets", "Beauty"];
+export const categories = ["Restaurant", "Shopping", "Markets", "Beauty"];
```

Comment:
We should probably put these in Amplify next quarter

Reply:
I think we have an enum for them in amplify idk why i still have this here